engines/sci/sound/drivers/cms.cpp

```diff
@@ -272,8 +272,9 @@ void CMSVoice::sendFrequency() {
 }
 
 void CMSVoice::cmsWrite(uint8 reg, uint8 val) {
-	_cms->portWrite(0x221 + _portOffset, reg);
-	_cms->portWrite(0x220 + _portOffset, val);
+	/* TODO: Call _cms->writeReg instead? */
+	_cms->write(0x221 + _portOffset, reg);
+	_cms->write(0x220 + _portOffset, val);
```

Comment:
Are you going to change this to writeReg as part of this PR? ScummVM does not need to use I/O ports of course; would be nice to get rid of them.

Reply:
OK, done.